docs/ja/formats.md

```diff
@@ -0,0 +1,328 @@
+<!--
+Copyright © 2012-2023 jrnl contributors
+License: https://www.gnu.org/licenses/gpl-3.0.html
+-->
+
+# フォーマット
+
+`jrnl`は様々な代替フォーマットをサポートしています。これらは、ジャーナルを`jrnl`のデフォルトとは異なる方法で表示したり、ジャーナルからのデータを他のプログラムにパイプして報告書を作成したり、`jrnl`のデータを好きなように使用したりするのに利用できます。
+
+これらのフォーマットはいずれも検索と組み合わせて使用でき（例：`jrnl -contains "lorem ipsum" --format json`）、検索結果を指定されたフォーマットで表示したり、単独で使用して（例：`jrnl --format json`）選択されたジャーナルのすべてのエントリーを表示したりできます。
+
+このページでは、すべての組み込みフォーマットの例を示していますが、`jrnl`はプラグインを通じてさらにフォーマットを追加することをサポートしているため、システムによってはさらに多くのフォーマットが利用可能な場合があります。システムで利用可能なフォーマットのリストについては、`jrnl --help`を参照してください。
+
+これらのフォーマットはどれも互換性があり、以下では便宜上「表示」、「データ」、「レポート」フォーマットにグループ分けしているだけです。
+
+## 表示フォーマット
+
+これらのフォーマットは主にターミナルでジャーナルを表示することを目的としています。それでも、他のフォーマットと同じように使用できます（選択すればファイルに書き込むこともできます）。
+
+### Pretty
+
+```sh
+jrnl --format pretty
+# または
+jrnl -1 # 任意の検索
+```
+
+これは`jrnl`のデフォルトフォーマットです。`--format`が指定されていない場合、`pretty`が使用されます。
+
+各エントリーのタイムスタンプをユーザー設定に従ってフォーマットし、同じ行にタイトルを表示します。その後、エントリーの本文が下に表示されます。
+
+このフォーマットは、設定ファイルの以下の値を通じて設定可能です（詳細は[Advanced Usage](./advanced.md)を参照）：
+
+- `colors`
+  - `body`
+  - `date`
+  - `tags`
+  - `title`
+- `indent_character`
+- `linewrap`
+- `timeformat`
+
+**出力例**:
+
+```sh
+2020-06-28 18:22 これは最初のサンプルエントリーです
+| これは最初のサンプルエントリーの本文テキストです。
+
+2020-07-01 20:00 これは2番目のサンプルエントリーです
+| これは2番目のサンプルエントリーの本文テキストですが、
+| これには@tagがあります。
+
+2020-07-02 09:00 これは3番目のサンプルエントリーです
+| これは3番目のサンプルエントリーの本文テキストです。
+```
+
+### Short
+
+```sh
+jrnl --format short
+# または
+jrnl --short
+```
+
+これはエントリーを短縮して、日付とタイトルのみを表示します。本質的には`pretty`フォーマットから各エントリーの本文を除いたものです。長いジャーナルエントリーがあり、検索に一致するエントリーのリストのみを表示したい場合に便利です。
+
+**出力例**:
+
+```sh
+2020-06-28 18:22 これは最初のサンプルエントリーです
+2020-07-01 20:00 これは2番目のサンプルエントリーです
+2020-07-02 09:00 これは3番目のサンプルエントリーです
+```
+
+### Fancy (または Boxed)
+
+```sh
+jrnl --format fancy
+# または
+jrnl --format boxed
+```
+
+このフォーマットは各エントリーを枠線で囲みます。これにより、各エントリーの開始と終了がわかりやすくなります。フォーマットがいかに自由形式であるかを示す例であり、また、そういったものが好きな人にとっては~_~ファンシー~_~に見えるかもしれません。
+
+!!! note
+日本語の場合、文字幅の問題でボックスが崩れてしまうようです。
+
+**出力例**:
+
+```sh
+┎──────────────────────────────────────────────────────────────────────╮2020-06-28 18:22
+┃ これは最初のサンプルエントリーです                                    ╘═══════════════╕
```

Comment:
For whatever reason, each of these three example output displays appears to be about 5 characters short of the visual "box" alignment. (The Japanese does have fewer characters than the English, but that doesn't seem to account for the difference.)

Reply:
When running jrnl with Japanese text, the box formatting breaks as shown in the attachment.
![result-jrnl](https://github.com/user-attachments/assets/9a9d86cc-f35c-4a98-af3b-31af36e0cec3)